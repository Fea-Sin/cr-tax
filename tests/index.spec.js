import React from 'react';
import { mount, render } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import Comp from '../src';


describe('cr-tax', () => {
  it('base test comp', () => {
    const wrapper = mount(
      <Comp />
    )
    expect('COMP IS DONE').toBe('COMP IS DONE')
  })
  it('snapshot to match', () => {
    const wrapper = render(
      <Comp />
    )
    expect( renderToJson(wrapper) ).toMatchSnapshot()
  })
})
