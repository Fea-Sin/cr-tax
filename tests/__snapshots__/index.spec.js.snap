// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cr-tax snapshot to match 1`] = `
<div
  class="cr-tax cr-tax-con"
>
  <div
    style="position:relative;overflow:hidden;width:100%;height:675px"
  >
    <div
      style="position:absolute;top:0;left:0;right:0;bottom:0;overflow:scroll;-webkit-overflow-scrolling:touch;margin-right:0;margin-bottom:0"
    >
      <div
        class="cr-tax-main"
      >
        <div
          class="cr-tax-top"
        >
          <div
            class="ant-row"
          >
            <div
              class="ant-col ant-col-8"
            >
              <div
                class="cr-tax-top-left"
              >
                <div
                  class="cr-tax-top-left-box"
                >
                  <span>
                    *发票代码:
                  </span>
                  <span
                    class="cr-tax-top-left-box-input"
                  >
                    <input
                      class="cr-tax-input"
                      disabled=""
                      value=""
                    />
                  </span>
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-8"
            >
              <div
                class="cr-tax-top-titleBox"
              >
                <div
                  class="cr-tax-top-titleBox-title"
                >
                  增值税专用发票
                </div>
                <div
                  class="cr-tax-top-titleBox-line"
                />
                <div
                  class="cr-tax-top-titleBox-sub"
                >
                  发票联
                </div>
              </div>
            </div>
            <div
              class="ant-col ant-col-8"
            >
              <div
                class="cr-tax-top-right"
              >
                <div
                  class="cr-tax-top-right-box"
                >
                  <span>
                    发票号码:
                  </span>
                  <span
                    class="cr-tax-top-right-box-input"
                  >
                    <input
                      class="cr-tax-input"
                      disabled=""
                      value=""
                    />
                  </span>
                </div>
                <div
                  class="cr-tax-top-right-box"
                >
                  <span>
                    开票日期:
                  </span>
                  <span
                    class="cr-tax-top-right-box-input"
                  >
                    <input
                      class="cr-tax-input"
                      disabled=""
                      value=""
                    />
                    <i
                      aria-label="icon: calendar"
                      class="anticon anticon-calendar cr-tax-top-right-box-calender"
                    >
                      <svg
                        aria-hidden="true"
                        class=""
                        data-icon="calendar"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                        />
                      </svg>
                    </i>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="cr-tax-dataBox"
        >
          <div
            class="cr-tax-dataBox-purchaser"
          >
            <table
              class="cr-tax-dataBox-table"
            >
              <tbody>
                <tr>
                  <td
                    class="cr-tax-dataBox-table-title"
                  >
                    购货单位
                  </td>
                  <td>
                    <div
                      style="margin-top:5px"
                    >
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          <span
                            class="cr-tax-dataBox-table-cellBox-star"
                          >
                            名
                          </span>
                          称
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          纳税人识别号
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          地址、电话
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          开户行及账户
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input cr-tax-dataBox-noLine"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                    </div>
                  </td>
                  <td
                    class="cr-tax-dataBox-table-title"
                  >
                    密码区
                  </td>
                  <td
                    class="cr-tax-dataBox-table-blank"
                  />
                </tr>
              </tbody>
            </table>
          </div>
          <div
            class="cr-tax-dataBox-data cr-tax-dataBox-interval"
          >
            <table
              class="cr-tax-dataBox-table cr-tax-dataBox-data-innerTable"
            >
              <thead>
                <tr
                  class="cr-tax-dataBox-data-tr"
                >
                  <th
                    class="cr-tax-dataBox-data-HW cr-tax-dataBox-data-tr-start"
                  >
                    *货物或应税劳务、服务名称
                  </th>
                  <th
                    class="cr-tax-dataBox-data-GG"
                  >
                    规格型号
                  </th>
                  <th
                    class="cr-tax-dataBox-data-DW"
                  >
                    单位
                  </th>
                  <th
                    class="cr-tax-dataBox-data-SL"
                  >
                    数量
                  </th>
                  <th
                    class="cr-tax-dataBox-data-DJ"
                  >
                    单价
                  </th>
                  <th
                    class="cr-tax-dataBox-data-JE"
                  >
                    *金额
                  </th>
                  <th
                    class="cr-tax-dataBox-data-SHL"
                  >
                    *税率
                  </th>
                  <th
                    class="cr-tax-dataBox-data-SE cr-tax-dataBox-data-tr-end"
                  >
                    税额
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr
                  class="cr-tax-dataBox-data-dataTr"
                >
                  <td
                    class="cr-tax-dataBox-data-dataTr-start"
                  >
                    <span
                      class="cr-tax-dataBox-data-dataTr-text"
                    />
                  </td>
                  <td />
                  <td />
                  <td />
                  <td />
                  <td />
                  <td>
                    <div>
                       
                    </div>
                  </td>
                  <td
                    class="cr-tax-dataBox-data-SE-Data cr-tax-dataBox-data-dataTr-end"
                  />
                </tr>
              </tbody>
            </table>
          </div>
          <div
            class="cr-tax-dataBox-seller"
          >
            <table
              class="cr-tax-dataBox-table cr-tax-dataBox-interval"
            >
              <tbody>
                <tr
                  class="cr-tax-dataBox-seller-tr"
                >
                  <td
                    class="cr-tax-dataBox-seller-HJBT cr-tax-dataBox-center"
                  >
                    <span>
                      价税合计
                    </span>
                    <span
                      style="font-size:13px"
                    >
                      （大写）
                    </span>
                  </td>
                  <td
                    class="cr-tax-dataBox-seller-HJB"
                  />
                  <td
                    class="cr-tax-dataBox-seller-HJST cr-tax-dataBox-center"
                  >
                    <span
                      style="font-size:13px"
                    >
                      （小写）
                    </span>
                  </td>
                  <td
                    class="cr-tax-dataBox-seller-HJS"
                  >
                    ¥undefined
                  </td>
                </tr>
              </tbody>
            </table>
            <table
              class="cr-tax-dataBox-table cr-tax-dataBox-interval"
            >
              <tbody>
                <tr>
                  <td
                    class="cr-tax-dataBox-table-title"
                  >
                    销货单位
                  </td>
                  <td>
                    <div
                      style="margin-top:5px"
                    >
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          <span
                            class="cr-tax-dataBox-table-cellBox-star"
                          >
                            名
                          </span>
                          称
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          纳税人识别号
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          地址、电话
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                      <div
                        class="cr-tax-dataBox-table-cellBox"
                      >
                        <span
                          class="cr-tax-dataBox-table-cellBox-title"
                        >
                          开户行及账户
                        </span>
                        <span>
                          <input
                            class="cr-tax-dataBox-input cr-tax-dataBox-noLine"
                            disabled=""
                            value=""
                          />
                        </span>
                      </div>
                    </div>
                  </td>
                  <td
                    class="cr-tax-dataBox-table-title"
                  >
                    备注
                  </td>
                  <td
                    class="cr-tax-dataBox-table-BZ"
                  >
                    <span
                      style="color:#3E7AFA"
                    />
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
    <div
      style="position:absolute;height:6px;display:none;right:2px;bottom:2px;left:2px;border-radius:3px"
    >
      <div
        style="position:relative;display:block;height:100%;cursor:pointer;border-radius:inherit;background-color:rgba(0,0,0,.2)"
      />
    </div>
    <div
      style="position:absolute;width:6px;display:none;right:2px;bottom:2px;top:2px;border-radius:3px"
    >
      <div
        style="position:relative;display:block;width:100%;cursor:pointer;border-radius:inherit;background-color:rgba(0,0,0,.2)"
      />
    </div>
  </div>
  <div
    class="cr-tax-footer"
  >
    <div>
      <span>
        认证状态:
      </span>
      <span
        class="cr-tax-footer-status"
        style="margin-left:8px"
      >
        <div
          class="ant-radio-group ant-radio-group-outline"
        >
          <label
            class="ant-radio-wrapper ant-radio-wrapper-disabled"
          >
            <span
              class="ant-radio ant-radio-disabled"
            >
              <input
                class="ant-radio-input"
                disabled=""
                type="radio"
                value="1"
              />
              <span
                class="ant-radio-inner"
              />
            </span>
            <span>
              已认证
            </span>
          </label>
          <label
            class="ant-radio-wrapper ant-radio-wrapper-checked ant-radio-wrapper-disabled"
            style="margin-left:0"
          >
            <span
              class="ant-radio ant-radio-checked ant-radio-disabled"
            >
              <input
                checked=""
                class="ant-radio-input"
                disabled=""
                type="radio"
                value="2"
              />
              <span
                class="ant-radio-inner"
              />
            </span>
            <span>
              未认证
            </span>
          </label>
        </div>
      </span>
      <span
        class="cr-tax-footer-ticketDate"
      >
        <span>
          认证日期:
        </span>
        <span
          class="cr-tax-top-right-box-input"
        >
          <input
            class="cr-tax-input cr-tax-footer-input"
            disabled=""
            value=""
          />
          <i
            aria-label="icon: calendar"
            class="anticon anticon-calendar cr-tax-top-right-box-calender"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="calendar"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
              />
            </svg>
          </i>
        </span>
      </span>
    </div>
  </div>
</div>
`;
